lib/src/main/java/io/ably/lib/http/TokenAuth.java

```diff
@@ -36,13 +36,13 @@ public void setTokenDetails(TokenDetails tokenDetails) {
 		this.encodedToken = Base64Coder.encodeString(tokenDetails.token).replace("=", "");
 	}
 
-	public TokenDetails authorise(AuthOptions options, TokenParams params) throws AblyException {
+	public boolean authorise(AuthOptions options, TokenParams params, boolean force) throws AblyException {
```

Comment:
Actually it is still called with `force=true`, with the `authOptions` otherwise being the stored defaults, from `Auth.renew()`, which is called in response to a server-initiated renewal.

> I guess this should have a .storedValues() as well.

This is wrong also, because that would remove any specified `queryTime` value.

So I don't propose any change for this.